app/lib/server/functions/sendMessage.js

```diff
@@ -198,6 +198,10 @@ export const sendMessage = function(user, message, room, upsert = false) {
 			}, message);
 			message._id = _id;
 		} else {
+			const messageAlreadyExists = message._id && Messages.findOneById(message._id);
```

Comment:
```suggestion
			const messageAlreadyExists = message._id && Messages.findOneById(message._id, { fields: { _id: 1 } });
```
this would avoid returning not used fields